export.py

```diff
@@ -170,24 +245,27 @@ def combine(all_posts, all_comments):
     os.makedirs('posts-html', exist_ok=True)
     os.makedirs('posts-markdown', exist_ok=True)
     os.makedirs('comments-markdown', exist_ok=True)
+    os.makedirs('posts-and-comments-markdown', exist_ok=True)
 
     posts_comments = group_comments_by_post(all_comments)
 
     for json_post in all_posts:
         id = json_post['id']
         jitemid = int(id) >> 8
 
-        date = datetime.strptime(json_post['date'], '%Y-%m-%d %H:%M:%S')
-        subfolder = '{0.year}-{0.month:02d}'.format(date)
+        date = datetime.strptime(json_post['eventtime'], '%Y-%m-%d %H:%M:%S')
+        subfolder = '{0.year}-{0.month:02d}-{0.day:02d}'.format(date)
```

Comment:
I would rather not apply this change to the codebase. I believe, only a small fraction of authors is so prolific that they need to group posts by day, not by month.